src/Microsoft.AspNetCore.Server.Kestrel/WebHostBuilderKestrelExtensions.cs

```diff
@@ -28,6 +29,7 @@ public static IWebHostBuilder UseKestrel(this IWebHostBuilder hostBuilder)
             return hostBuilder.ConfigureServices(services =>
             {
                 services.AddTransient<IConfigureOptions<KestrelServerOptions>, KestrelServerOptionsSetup>();
+                services.AddTransient<IConfigureOptions<KestrelServerOptions>, KestrelServerConfigureOptions>();
```

Comment:
That works, the other alternative would be to register

`services.AddTransient<IConfigureOptions<KestrelServerOptions>, ConfigureDefaults<KestrelServerConfigureOptions>()`

Which would let us just remove this line once the DI is fixed.

For preview 2 I don't think it really matters which one we do.  Its slightly easier to cleanup if we go with lining up all the defaults for now.